climada/test/test_blackmarble.py

```diff
@@ -74,7 +74,7 @@ def test_sint_maarten_pass(self):
                       cm.output[0])
         self.assertIn("Processing country Sint Maarten.", cm.output[1])
         self.assertIn("Generating resolution of approx 0.2 km.", cm.output[2])
-        self.assertAlmostEqual(ent.gdf.value.sum(), 3.658e+08 * (4 + 1))
+        self.assertTrue(np.isclose(ent.gdf.value.sum(), 1.023e+09 * (4 + 1), 1))
```

Comment:
Are you sure that this is what you want? You set `rtol=1` in the `numpy.isclose` function. That means that the first value may deviate from the second by 100%. For example, `np.isclose(0, 1000, 1)` is True. Same for `np.isclose(2000, 1000, 1)`.

Reply:
Thanks, changed it ad also above which was there from before. Also changed the rest.